migrations/transformWordpressToUrlSlug.js

```diff
@@ -21,7 +21,11 @@ module.exports = function (migration) {
       }
 
       const wordPressUrlValue = wordPressUrl[currentLocale].toString();
-      const newSlug = wordPressUrlValue.match(/\/\/([\S]+)/)[1];
+      console.log("WordPressURL: " + wordPressUrlValue);
+
+      const urlArray = wordPressUrlValue.split("/");
+      const newSlug = urlArray[urlArray.length - 2]; // there is a trailing '/' on wordpress URL
```

Comment:
If this doesn't work so well, maybe I'd suggest just a regex match instead?

`/2020/01/something/` or `/2020/01/something` should match this, but this is assuming dates:
```js
const slugMatch = wordPressUrlValue.match(/^\/[\d]+\/[\d]+\/([A-z-_0-9]+)\/?$/)
const newSlug = slugMatch ? slugMatch[1] : undefined;
```

You can test this in your browser console if you want:
```js
var str = `/2020/01/some-random-url/`;
str.match(/^\/[\d]+\/[\d]+\/([A-z-_0-9]+)\/?$/)[1]
```